crates/macro-support/src/lib.rs

```diff
@@ -21,6 +21,16 @@ use syn::parse::{Parse, ParseStream, Result as SynResult};
 
 mod parser;
 
+#[cfg(feature = "strict-macro")]
+const _: () = {
+    #[deprecated(
+        note = "strict-macro feature is deprecated and will be removed in a future version"
+    )]
+    const WASM_BINDGEN_STRICT_MACRO: () = {};
+
+    WASM_BINDGEN_STRICT_MACRO
+};
+
```

Comment:
I don't think that this will doing anything for consumers of the crate because warnings like this, for the crate itself, are silenced when building upstream crates.io crates.

Reply:
Sigh, you're right, of course. Once again I wish Rust just had a `compile_warning!`... I'll remove this code. I don't suppose there's any other reliable way to "deprecate" Cargo features?